src/v2/ion/IonResponseHelper.js

```diff
@@ -1,47 +1,70 @@
 import { _ } from 'okta';
 
-/*
-returns errors
-errors = [
-  {property : fieldName1, errorSummary: [errorMessage1]},
-  {property : fieldName2, errorSummary: [errorMessage1]}
-  {property : fieldName3, errorSummary: [errorMessage1, errorMessage2]}
-]
-*/
+const convertErrorMessageToErrorSummary = (maybeFormName, remediationValues = []) => {
```

Comment:
why this name `maybeFormName` ?

Reply:
cause this function has been used at several places but the formName is null at one case.
yeah, now I see it's confusing. The param name shall be `formName` but just it's type is `Maybe String`.